packages/validation/test/index.js

```diff
@@ -40,6 +40,6 @@ describe("validation", () => {
       return errorsToString(errors);
     };
 
-    compareWithExpected(testSuites, pre, "throws.txt");
+    compareWithExpected(testSuites, pre, "output.txt");
```

Comment:
I would suggest keeping throws for parser/early errors since it's builin in the "test framework"